docker/openmower_entrypoint.sh

```diff
@@ -13,10 +13,19 @@ fi
 
 if [[ "$MODE" == "osv1" ]]; then
     source /config/mower_config.sh
+    # If OM_V2 is set (to anything), set HARDWARE_PLATFORM=2, else 1
+    if [[ -n ${OM_V2+x} ]]; then
```

Comment:
Testing OM_V2 to anything looks dangerous to me :innocent:

I would go with something like: `[[ "${OM_V2,,}" =~ ^(true|1|yes)$ ]]`

Reply:
fixed in #05d123b